website/scripts/javadocs.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
-#set -e TODO: figure out why this breaks CI and re-enable (https://github.com/twitter/heron/issues/766)
-
+set -e
```

Comment:
There's someone going on with the way that docs generation script is trying to build with bazel here that is making the protobuf gen fail. I suspect we need to change the bazel command that we execute somehow.